<robot name="floaty">
    <link name="base_link"/>
    <link name="camera_tilt_link"/>
    <link name="camera_link">
        <visual>
            <origin rpy="0 0 0" xyz="0 0 0" />
            <geometry>
                <mesh filename="package://gebsyas/meshes/camera.dae" />
            </geometry>
        </visual>
        <collision>
            <origin rpy="0 0 0" xyz="0 0 0" />
            <geometry>
                <mesh filename="package://gebsyas/meshes/camera.dae" />
            </geometry>
        </collision>
    </link>

    <joint name="camera_lift_joint" type="prismatic">
        <parent link="base_link"/>
        <child  link="camera_tilt_link"/>
        <axis xyz="0 0 1"/>
        <limit lower="0.9" upper="1.5" effort="100" velocity="0.5"/>
    </joint>

    <joint name="camera_tilt_joint" type="revolute">
        <parent link="camera_tilt_link"/>
        <child  link="camera_link"/>
        <axis xyz="0 1 0"/>
        <limit lower="-1.2" upper="1.2" effort="100" velocity="0.5"/>
    </joint>
</robot>